widget/embedded/src/constants/errors.ts

```diff
@@ -10,26 +10,30 @@ export const errorMessages = () => {
     liquiditySourcesError: {
       title: i18n.t('Please reset your liquidity sources.'),
       description: i18n.t(
-        'You have limited the liquidity sources and this might result in Rango finding no routes. Please consider resetting your liquidity sources'
+        'You have limited the liquidity sources and this might result in Rango finding no routes. Please consider resetting your liquidity sources.'
       ),
     },
     noResultError: {
-      title: i18n.t('No Routes Found'),
+      title: i18n.t('No Routes Found.'),
       description: i18n.t(
         "Reasons why Rango couldn't find a route: low liquidity on token, very low input amount or no routes available for the selected input/output token combination."
       ),
     },
     bridgeLimitErrors: {
-      increaseAmount: i18n.t('Bridge Limit Error: Please increase your amount'),
-      decreaseAmount: i18n.t('Bridge Limit Error: Please decrease your amount'),
+      increaseAmount: i18n.t(
+        'Bridge Limit Error: Please increase your amount.'
+      ),
+      decreaseAmount: i18n.t(
+        'Bridge Limit Error: Please decrease your amount.'
+      ),
     },
     highValueLossError: {
       impactTitle: i18n.t('High Price Impact'),
       title: i18n.t('Price impact is too high!'),
       description: i18n.t(
         'The price impact is significantly higher than the allowed amount. If you are sure, continue, otherwise, change the swap.'
       ),
-      confirmMessage: i18n.t('Confirm High price impact'),
+      confirmMessage: i18n.t('Confirm high price impact'),
```

Comment:
Can you check NaN value here?
You can test it with this route:
http://localhost:3002/?fromBlockchain=ETH&fromToken=USDT--0xdac17f958d2ee523a2206206994597c13d831ec7&toBlockchain=AVAX_CCHAIN&toToken=USDT.E--0xc7198437980c041c805a1edcba50c1ce5db95118&fromAmount=11111

![Screenshot from 2023-11-21 15-48-37](https://github.com/rango-exchange/rango-client/assets/99180934/f09c9628-d96b-49fe-8073-e22678250f6e)